ingest-app/src/cmr/ingest/services/ingest_service.clj

```diff
@@ -397,6 +397,22 @@
           (dissoc :revision-date :transaction-id)
           (update-in [:revision-id] inc)))))
 
+(defn delete-variable
+  "Deletes a tag with the given concept id"
+  [context variable-key]
+  (let [existing-concept (fetch-variable-concept context variable-key)]
+    (mdb/save-concept
+      context
+      (-> existing-concept
+          ;; Remove fields not allowed when creating a tombstone.
+          (dissoc :metadata :format :provider-id :native-id :transaction-id)
```

Comment:
This was carried over from tags, I believe ... /me looks it up ... yup; just confirmed. Does removing that data make sense for tags but not variables? Perhaps @ygliuvt knows?